storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/queries/Query.java

```diff
@@ -477,6 +477,50 @@ public CompleteBuilder limit(@Nullable String limit) {
         }
 
         /**
+         * Optional: Specifies {@code LIMIT} clause.
+         * <p>
+         * Optional specifier that limits the number of rows returned by the query.
+         *
+         * @param limit positive number of rows returned by the query.
+         * @return builder.
+         * @see Query#limit()
+         */
+        @NonNull
+        public CompleteBuilder limit(final int limit) {
+            if (limit <= 0) {
+                throw new IllegalStateException("Limit should have a positive value");
```

Comment:
Can we please add `limit` to the exception message? So it will be much easier to see the problem!